src/compiler/checker.ts

```diff
@@ -4671,6 +4688,10 @@ namespace ts {
                 if (symbol.flags & SymbolFlags.Prototype) {
                     return links.type = getTypeOfPrototypeProperty(symbol);
                 }
+                // CommonsJS require/module/exports all have type any.
+                if (symbol === requireSymbol || symbol === moduleSymbol || symbol === exportsSymbol) {
+                    return links.type = anyType;

```

Comment:
type of export is not `any`, it is the type of the module object. @andy-ms had a related change a while back, we should consolidate the two.. https://github.com/Microsoft/TypeScript/pull/17745

Reply:
Oops. Thanks for catching that.